.github/workflows/lint.yml

```diff
@@ -0,0 +1,20 @@
+name: Lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: Run rubocop
```

Comment:
Can you change this to `Run RuboCop` to have the correct capitalization for RuboCop?

Reply:
done, I have updated the PR